ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -766,7 +766,6 @@
 						TestTargetID = 2D02E47A1E0B4A5D006451C7;
 					};
 					888DAA1429B48C8900311304 = {
-						DevelopmentTeam = J22M3M9VYU;
 						ProvisioningStyle = Automatic;
 					};
 					889CAB5E27D46CD6007F5892 = {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `DevelopmentTeam` has been removed from the project settings. This could potentially cause issues with code signing and app distribution if not properly managed in your CI/CD pipeline or manually before distribution.
```diff
- DevelopmentTeam = J22M3M9VYU;
+ // No development team specified
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->